### PR TITLE
Issue #95 - Filter webcompat.com see also bugs from Bugzilla result counts

### PR DESCRIPTION
**Pull request by miketaylr**
(These bugs will be captured in the Duplicates column value)

Still tweaking the bugzilla queries, this commit just gets the right numbers.

### CODE REVIEW

**Comment by miketaylr** [comment]
(will paste a test run in a few so people can click on things and sanity-check)

**Comment by miketaylr** [comment]
https://docs.google.com/spreadsheets/d/15mfn1beIFGJoiqrzDgVQcrg8NQ-KwcK2tdpOtxXIM4M/edit#gid=1620202815

File: bugs.js [hunk 1]
```diff
@@ -104,15 +104,15 @@ const getBugzilla = async (website, bugzillaKey, minDate, maxDate = new Date())
...
-    const resolvedApiQuery = `https://bugzilla.mozilla.org/rest/bug?include_fields=id,summary,status,priority,product,component,creator,op_sys${helpers.getBugzillaPriorities()}${helpers.getBugURL(website)}&chfield=bug_status&chfieldfrom=${maxDateQuery}&chfieldvalue=RESOLVED&f1=OP&f3=creation_ts&f4=creation_ts&o3=greaterthaneq&o4=lessthaneq${helpers.getBugzillaProducts()}&v3=${minDateQuery}&v4=${maxDateQuery}&api_key=${bugzillaKey}${searchConstraintQueryFragment}`;
+    const openQuery = `https://bugzilla.mozilla.org/buglist.cgi?query_format=advanced&f1=OP${helpers.getBugzillaPriorities()}${helpers.getBugURL(website)}&o2=greaterthaneq&list_id=14636479&v2=${minDateQuery}&resolution=---&f2=creation_ts${helpers.getBugzillaStatuses()}${helpers.getBugzillaProducts()}${maxDateQueryFragment(3)}${searchConstraintQueryFragment}&f4=CP&f5=OP&v6=webcompat.com&f6=see_also&o6=notsubstring&f7=see_also&o7=notsubstring&v7=web-bugs`;
```
Comment (past) [inline bugs.js]: This would benefit from another helper like `maxDateQueryFragment(position)` just to cut down on the copy-pasta.

File: bugs.js [hunk 2]
```diff
@@ -104,15 +104,15 @@ const getBugzilla = async (website, bugzillaKey, minDate, maxDate = new Date())
...
+    const resolvedApiQuery = `https://bugzilla.mozilla.org/rest/bug?include_fields=see_also,id,summary,status,priority,product,component,creator,op_sys${helpers.getBugzillaPriorities()}${helpers.getBugURL(website)}&chfield=bug_status&chfieldfrom=${maxDateQuery}&chfieldvalue=RESOLVED&f1=OP&f3=creation_ts&f4=creation_ts&o3=greaterthaneq&o4=lessthaneq${helpers.getBugzillaProducts()}&v3=${minDateQuery}&v4=${maxDateQuery}&api_key=${bugzillaKey}${searchConstraintQueryFragment}`;
     let openResults = await helpers.bugzillaRetry(openApiQuery);
     let resolvedResults = await helpers.bugzillaRetry(resolvedApiQuery);
-    openResults = openResults.bugs.filter(helpers.isNotQA);
-    resolvedResults = resolvedResults.bugs.filter(helpers.isNotQA);
+    openResults = openResults.bugs.filter(helpers.isNotQA).filter(helpers.filterWebCompatSeeAlso);
```
Comment (past) [inline bugs.js]: I suppose there is a tiny chance that someone will update the see-also field between the `getBugzilla` and `getDuplicates`, but let's not worry about it :-)

File: bugs.js (hunk 1 above)
Comment (miketaylr) [reply bugs.js]: Good point, we can do that.

**Comment by hackhnaton1994** [comment]
Help wanted Link To tool Dev

File: bugs.js (hunk 2 above)
Comment (hackhnaton1994) [reply bugs.js]: When u change any Issue....Will start again computer they still Have determine

**Pull request merged**